apisix/discovery/eureka.lua

```diff
@@ -157,7 +157,7 @@ local function parse_instance(instance)
     local ip = instance.ipAddr
     if not ipmatcher.parse_ipv4(ip) and
             not ipmatcher.parse_ipv6(ip) then
-        log.error("invalid ip:", ip)
+        log.error(instance.app, " service ", instance.hostName, " node IP ", ip, " is invalid(must be IPv4 or IPv6).")
```

Comment:
this line is too long. 
you can rebase your branch with master branch and run `make lint`, it show you similar warning message.